src/components/OrganizationSaas/Dashboard/SelectCustom.tsx

```diff
@@ -0,0 +1,82 @@
+import { FormControl, FormErrorMessage, FormLabel } from '@chakra-ui/react'
+import { ChakraStylesConfig, GroupBase, Select } from 'chakra-react-select'
+import { Controller, useFormContext } from 'react-hook-form'
+import { useTranslation } from 'react-i18next'
+import useDarkMode from '~src/themes/saas/hooks/useDarkMode'
+
+export interface SelectCustomProps {
+  options: { label: string; value: string }[]
+  formValue: string
+  label?: string
+  placeholder?: string
+  required?: boolean
+}
+
+const customStyles: ChakraStylesConfig<any, false, GroupBase<any>> = {
+  control: (provided) => ({
+    ...provided,
+    borderColor: 'gray.200',
+    minH: '50px',
+    borderRadius: 'xl',
+    _hover: {
+      borderColor: 'gray.300',
+    },
+    _focus: {
+      borderColor: 'gray.300',
+      boxShadow: 'none',
+    },
+  }),
+}
+
+const SelectCustom = ({
+  options,
+  formValue,
+  label,
+  placeholder = 'Choose an option',
+  required = false,
+}: SelectCustomProps) => {
```

Comment:
Imo this component have to be moved inside the `src/components/Layout` folder. Is a Layout generic component used on different places of the code. 

And more, provably it can be merged with the `SaasSelector.tsx`, and put the custom selectors + the generic SelectCustom component all in same file.